lib/util.js

```diff
@@ -504,49 +505,60 @@ const isArray = exports.isArray = Array.isArray;
 function isBoolean(arg) {
   return typeof arg === 'boolean';
 }
-exports.isBoolean = isBoolean;
+exports.isBoolean = deprecate(isBoolean
+  , 'util.is* functions are deprecated, please use a user-land alternative.');
```

Comment:
Would it be better for these to write it out for each, e.g., `util.isBoolean is deprecated, please use a user-land alternative.`?